FirebaseSessions/Sources/SessionStartEvent.swift

```diff
@@ -109,7 +109,7 @@ class SessionStartEvent: NSObject, GDTCOREventDataObject {
                                    appInfo: ApplicationInfoProtocol) {
     switch subscriber {
     case .Performance:
-      proto.application_info.apple_app_info.mcc_mnc = makeProtoString(appInfo.mccMNC)
+      proto.application_info.apple_app_info.mcc_mnc = makeProtoString("")
```

Comment:
Do we know what happens in the Backend when there is no radio information? Hopefully that does not drop the event due to validation failure?

Reply:
We never send a radio when just Crashlytics is installed, so this should not impact